index.html

```diff
@@ -3752,6 +3752,69 @@ <h3>Media Type Precision</h3>
           </p>
         </section>
 
+        <section>
+          <h2>JSON Processing</h2>
+
+          <p>
+While the base media types for this specification are JSON-LD documents, JSON-LD
+processing is not required to be performed in a variety of scenarios. These
+scenarios include, but are not limited to:
+          </p>
+
+          <ul>
+            <li>
+When using the JOSE or COSE specifications to secure a
+<a>verifiable credential</a> or <a>verifiable presentation</a>.
+            </li>
+            <li>
+When performing JSON Schema validation.
+            </li>
+            <li>
+When serializing or deserializing <a>verifiable credentials</a> or
+<a>verifiable presentations</a> into databases that index their contents.
+            </li>
+            <li>
+When operating on <a>verifiable credentials</a> or <a>verifiable
+presentations</a> in a software application after verification or validation
+is performed.
+            </li>
+            <li>
+When an application chooses to process the media type using the `+json`
+structured media type suffix.
+            </li>
+          </ul>
+
+          <p>
+That is, pure JSON processing is allowed as long as the document being
+consumed or produced is a <a>conforming document</a>. If pure JSON processing
+is desired, an implementer is advised to follow the following rule:
+          </p>
+
+          <ul>
+            <li>
+Ensure that all values associated with a `@context` property are in the
+expected order, the contents of the context files match known good
+cryptographic hashes for each file, and the contents are known to be valid.
+            </li>
+          </ul>
+
+          <p>
+Using static context files and a JSON Schema to ensure proper URL values and
+order is one acceptable approach when implementing the rule above.
```

Comment:
```suggestion
Using static context files with a JSON Schema is one acceptable approach when implementing the rule above. This ensures proper term identification, typing, and order when a JSON document is processed as JSON-LD.
```
or something like that

Reply:
```suggestion
Using static context files with a JSON Schema is one acceptable approach to
implementing the rule above. This can ensure proper term identification,
typing, and order, when a JSON document is processed as JSON-LD.
```